pkg/block/azure/adapter.go

```diff
@@ -644,6 +644,9 @@ func (a *Adapter) GetPresignUploadPartURL(_ context.Context, _ block.ObjectPoint
 func (a *Adapter) ListParts(_ context.Context, _ block.ObjectPointer, _ string, _ block.ListPartsOpts) (*block.ListPartsResponse, error) {
 	return nil, block.ErrOperationNotSupported
 }
+func (a *Adapter) ListMultipartUploads(_ context.Context, _ block.ObjectPointer) (*block.ListMultipartUploadsResponse, error) {
```

Comment:
@Isan-Rivkin I agree with @ItamarYuran's decision to implement this only for s3.
WDYT?